src/components/markdown/markdown.tsx

```diff
@@ -48,25 +49,42 @@ export class Markdown {
     public whitelist?: CustomElementDefinition[] =
         globalConfig.markdownWhitelist;
 
+    /**
+     * Enable lazy loading for images
+     */
+    @Prop()
+    public lazyLoadImages = false;
+
     @Watch('value')
     public async textChanged() {
         try {
             const html = await markdownToHTML(this.value, {
                 forceHardLineBreaks: true,
                 whitelist: this.whitelist ?? [],
+                lazyLoadImages: this.lazyLoadImages,
             });
+
             this.rootElement.innerHTML = html;
+
+            if (this.lazyLoadImages) {
+                this.imageLazyLoader.setupLazyLoading(this.rootElement);
```

Comment:
If we do something like this I think we can get rid of some of the comments the rabbit made :smile: :rabbit2: 
```suggestion
                this.imageLazyLoader = new ImageLazyLoader(this.rootElement);
```

Reply:
Good point, I will fix.

Reply:
[962fabf](https://github.com/Lundalogik/lime-elements/pull/3559/commits/962fabf95b625105dd7ed41db466eee9b2c07f0a)